ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Mock.hs

```diff
@@ -362,18 +362,27 @@ deriving instance OuroborosTag p => Show (LedgerState (SimpleBlock p c))
 -- This is needed in order to assign stake to nodes.
 type AddrDist = Map Addr NodeId
 
-instance (BftCrypto c, SimpleBlockCrypto c')
+instance (BftCrypto c, SimpleBlockCrypto c', Serialise (Payload (Bft c) (SimplePreHeader (Bft c) c')))
```

Comment:
It still compiles if you remove this new `Serialise (Payload ..)` constraint from this instance and the two instances below.